src/api/app/views/webui/users/notifications/_notifications_list.html.haml

```diff
@@ -9,40 +9,50 @@
           Mark notifications as "Read" and they'll appear here
         - else
           There are no notifications, but there's a world of opportunities!
-    - else
-      .text-center
-        %span.ml-3= page_entries_info notifications, entry_name: 'notification'
-        = link_to_all unless notifications.total_pages == 1 && params[:show_all].nil?
+- else
+  - update_path = my_notification_path(type: selected_filter[:type], project: selected_filter[:project],
+                                       page: params[:page], show_all: params[:show_all])
+  = form_tag(update_path, method: :put, remote: true) do
+    = render partial: 'notification_action_bar', locals: { type: selected_filter[:type] }
+    .card
+      .card-body
+        .text-center
+          %span.ml-3= page_entries_info notifications, entry_name: 'notification'
+          = link_to_all unless notifications.total_pages == 1 && params['show_all'].nil?
+        .list-group.list-group-flush.mt-3
+          - notifications.each do |n|
+            - notification = NotificationPresenter.new(n)
+            .list-group-item.px-0.px-md-1.py-3
+              .row
+                .col-auto
```

Comment:
To reduce the unneeded gap between the checkbox and the notification's content, I would suggest this:

```suggestion
                .col-auto.pr-0
```